src/creature.cpp

```diff
@@ -565,22 +566,22 @@ bool Creature::dropCorpse(Creature* lastHitCreature, Creature* mostDamageCreatur
 				break;
 		}
 
-		Tile* tile = getTile();
+		auto tile = getTile();
 
 		if (splash) {
 			g_game.internalAddItem(tile, splash, INDEX_WHEREEVER, FLAG_NOLIMIT);
 			g_game.startDecay(splash);
 		}
 
-		Item* corpse = getCorpse(lastHitCreature, mostDamageCreature);
+		auto corpse = getCorpse(lastHitCreature, mostDamageCreature);
```

Comment:
Could this be `const auto`
```suggestion
		const auto corpse = getCorpse(lastHitCreature, mostDamageCreature);
```